test/CMakeLists.txt

```diff
@@ -28,6 +22,17 @@ if(LIBCORO_FEATURE_NETWORKING)
     )
 endif()
 
+if(LIBCORO_FEATURE_THREADING)
+    list(APPEND LIBCORO_TEST_SOURCE_FILES
+        bench.cpp
+        test_semaphore.cpp
+        test_io_scheduler.cpp
+        test_ring_buffer.cpp
+        test_shared_mutex.cpp
+        test_thread_pool.cpp
```

Comment:
This file seems like it can be moved out of the feature? I don't see any usages of `coro::io_scheduler` in it.

Reply:
Thanks, done!